### PR TITLE
Fix errors in pqact files

### PR DESCRIPTION
**Pull request by tiffanycmeyer13**

### CODE REVIEW

File: rpms/awips2.upc/Installer.ldm/patch/etc/pqact.goesr [hunk 1]
```diff
@@ -74,7 +74,7 @@ NIMAGE	^/data/ldm/pub/native/satellite/GOES/([^/]*)/Products/CloudSnow/([^/]*)/(
 # NPP/VIIRS
 #
 # Polar products
-NOTHER	^(TI[A-W]...) (KNES) (......)
```
Comment (srcarter3) [inline rpms/awips2.upc/Installer.ldm/patch/etc/pqact.goesr:77]: This change is related to the VIIRS change and CIMSS color product, right?

File: rpms/awips2.upc/Installer.ldm/patch/etc/pqact.main [hunk 2]
```diff
@@ -697,6 +697,12 @@ HDS	^LDIZ1[1-7] KWNS (..)(..)(..)
 HDS	^LDIZ48 KWNS (..)(..)(..)
 	FILE	-edex -log -close
 	/awips2/data_store/spc/(\1:yyyy)(\1:mm)(\1:dd)(\1:hh).day4_outlook.grib2
+# Prob of Severe Thunderstorm Grids
```
Comment (srcarter3) [inline rpms/awips2.upc/Installer.ldm/patch/etc/pqact.main:700]: Where did this come from? Something that was in v18 previously, or somewhere else?

Comment (tiffanycmeyer13) [reply rpms/awips2.upc/Installer.ldm/patch/etc/pqact.main:700]: What I had added into the pqact.conf.priority that had the HSD ANY error, I had forgot to copy it to this main pqact

File: rpms/awips2.upc/Installer.ldm/patch/etc/pqact.goesr (hunk 1 above)
Comment (tiffanycmeyer13) [reply rpms/awips2.upc/Installer.ldm/patch/etc/pqact.goesr:77]: yes, to remove the SBN GOES Single Channel data

**Pull request merged**